index.html

```diff
@@ -4430,9 +4548,7 @@ <h2 id="AudioBuffer">
           <a><code>OfflineAudioContext</code></a> and an
           <a><code>AudioContext</code></a>.
         </p>
-        <dl title=
-        "[Constructor(&lt;a&gt;AudioBufferOptions&lt;a&gt; options)]interface AudioBuffer"
-        class="idl">
+        <dl title="interface AudioBuffer" class="idl">
```

Comment:
Those are not `AudioNode`s, the goal is to fix it in a different PR (this PR and issue is about AudioNodes, there are other issues about other objects).

This is different because the notion of "associated AudioContext" does not apply, among other things. I've reinstated this constructor.

Anyways, I plan on following up immediately with defining the ctors for other objects, this PR is big enough already.